controllers/event_rsvp.rb

```diff
@@ -71,7 +84,7 @@ def push_update(topic, prop)
     channel = "/calendar-events/#{topic.id}"
 
     msg = {
-      current_user_id: current_user.id,
+      current_user_name: current_user.username,
```

Comment:
@fzngagan This is sent to the client to preclude the current user from being updated via the messagebus (their client is already updated when they click the button). It doesn't need to be changed to a username.